remove_rke2.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+#
+# Copyright 2023 Justin Cook
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to
+# deal in the Software without restriction, including without limitation the
+# rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+# sell copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+# Remove multipass rke2 nodes configured from this code base.
+
+set -o errexit
+
+$(which kubectl) config delete-context generic-rke2-cluster || /usr/bin/true
```

Comment:
There seems to be a mismatch in naming of `rke-cluster` vs `rke2-cluster` between `multipass-rke2.sh` and `remove_rke2.sh`.

I think we are ready to merge after correcting that.

Reply:
Gosh sorry, a bit sloppy. I changed it to rke2 on context. Please note the remove_multipass.sh script is pure convenience and only works when NAME=generic.